src/styles/blocks/claendar.scss

```diff
@@ -0,0 +1,43 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $indantation;
+  margin: 1% auto;
+  padding: $paddings;
+  box-sizing: border-box;
+  max-width: $calendarWidth;
+  @for $index from 1 through 31 {
+    &__day-#{$index}::before {
+      content: "#{$index}";
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      font-size: 30px;
+      box-sizing: border-box;
+      background-color: #eee;
+      @extend %box;
+      border: 1px solid black;
+      border-radius: 5px;
+      font-family: Arial, Helvetica, sans-serif;
+      cursor: pointer;
+      transition: 0.5s;
+    }
+    & > :hover::before {
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
```

Comment:
Don't need to add all these styles for before pseudo-class. You can add it for day, and leave here just content propperty